python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -670,8 +711,15 @@ def _measure_aperture_correction(self, exposure, bright_sources):
             necessary for point source determination for the aperture correction
             calculation.
         """
-        result = self.measure_aperture_correction.run(exposure, bright_sources)
-        exposure.setApCorrMap(result.apCorrMap)
+        norm_ap_corr_map = exposure.info.getApCorrMap()
+        ap_corr_map = self.measure_aperture_correction.run(exposure, bright_sources).apCorrMap
+
+        # Need to merge the aperture correction map from the normalization.
+        if norm_ap_corr_map:
+            for key in norm_ap_corr_map:
+                ap_corr_map[key] = norm_ap_corr_map[key]
+
+        exposure.setApCorrMap(ap_corr_map)
```

Comment:
And you should be able to assign the property, too (I realize I didn't do that in the original, but I think I added it after I wrote this method):

```suggestion
        exposure.apCorrMap = ap_corr_map 
```